src/qt/guiutil.cpp

```diff
@@ -357,6 +367,21 @@ bool isObscured(QWidget *w)
         && checkPoint(QPoint(w->width() / 2, w->height() / 2), w));
 }
 
+void bringToFront(QWidget *w)
+{
+#ifdef Q_OS_MAC
+    id app = objc_msgSend((id) objc_getClass("NSApplication"), sel_registerName("sharedApplication"));
+    objc_msgSend(app, sel_getUid("activateIgnoringOtherApps:"), YES);
```

Comment:
Apple [documentation](https://developer.apple.com/documentation/appkit/nsapplication/1428468-activateignoringotherapps?language=objc) suggests that this normally shouldn't be needed, which suggests there's a deeper problem somewhere.

Reply:
I'd also like to know a bit more about this, and the new `<objc/objc-runtime.h>` #include.

Reply:
Sure.

Reply:
Added comment.

Reply:
According to the `sel_getUid` [documentation](https://developer.apple.com/documentation/objectivec/1418625-sel_getuid?language=objc):
> The implementation of this method is identical to the implementation of `sel_registerName`.

On master(750415701):
- `sel_registerName` used 4 times
- `sel_getUid` not used at all

Could `sel_getUid` replace with `sel_registerName`?